patches/net/minecraft/client/resources/model/ModelResourceLocation.java.patch

```diff
@@ -0,0 +1,24 @@
+--- a/net/minecraft/client/resources/model/ModelResourceLocation.java
++++ b/net/minecraft/client/resources/model/ModelResourceLocation.java
+@@ -8,6 +_,7 @@
+ @OnlyIn(Dist.CLIENT)
+ public record ModelResourceLocation(ResourceLocation id, String variant) {
+     public static final String INVENTORY_VARIANT = "inventory";
++    public static final String STANDALONE_VARIANT = "standalone";
+ 
+     public ModelResourceLocation(ResourceLocation id, String variant) {
+         variant = lowercaseVariant(variant);
+@@ -21,6 +_,13 @@
+ 
+     public static ModelResourceLocation inventory(ResourceLocation p_352141_) {
+         return new ModelResourceLocation(p_352141_, "inventory");
++    }
++
++    /**
++     * Construct a {@code ModelResourceLocation} for use in the {@link net.neoforged.neoforge.client.event.ModelEvent.RegisterAdditional}
```

Comment:
Maybe document the meaning of `standalone` (load the path given rather than passing through the blockstate or item processing first)?

Reply:
OK a few extra comments :D
- Should also mention somewhere (in the event?) that this is the MRL that must be used to recover the model from the `BakedModelManager`.
- You should end the sentences with full stops. :D